tests/contract.rs

```diff
@@ -0,0 +1,63 @@
+use cucumber::{ given, when, then, World };
```

Comment:
We have contract tests crate - move theses files there. 
`crates/contract-tests/tests/...`

Reply:
moved